### ISSUE TITLE
Scroll controler doesn't work 

### ISSUE THREAD

**Issue by nashyvan** [iametza/ckeditor-html5-audio#30]
**Сan not scroll audio for a certain time.**

**Comment by dannluciano** [comment]
@yariknashivan Can you solve this issue?

**Comment by nashyvan** [comment]
> @yariknashivan Can you solve this issue? 
> 
> 

yes, i solved it long time ago. the problem was in media paths in django urls.py and settings.py

**Comment by dannluciano** [comment]
@yariknashivan Can you help me in this?

**Comment by dannluciano** [comment]
@yariknashivan can you help me?

**Comment by nashyvan** [comment]
do you use django or another framework?
if django it helped me:

https://docs.djangoproject.com/en/3.1/howto/static-files/

**Comment by dannluciano** [comment]
@yariknashivan Yes I'm using Django. I can play the song but seek the audio don't work.
